### PR TITLE
Chore: Add docs

### PR DESCRIPTION
**Pull request by AkseliManninen**
## Description

Closes #[issue number]

- Add a doc about adding a custom domain. This can be useful later if we change the hosting platform.
- Small updates to README.md.
- We have our own domain working 🥳 

## Screenshot

image

### CODE REVIEW

File: docs/custom-domain.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+1. Navigate to Certificates in the fly website.
+2. Add new certificates for `lunssi.fi` and `www.lunssi.fi`.
+3. Check that the addresses get verified after adding info to Domainkeskus.
+
+## Domainkeskus
+
+1. Login to Domainkeskus.
+2. Navigate to Domains > Manage Domain > Manage DNS.
+3. Add A and AAA records for @ and www. The records can be found from fly.io.
```
Comment (yannjor) [inline docs/custom-domain.md:15]: I think this should be AAAA

**Review by yannjor — APPROVED** [review]
Awesome! We need to update the new domain also to the site metadata. You could maybe do it in the same PR or do a new one.

here: https://github.com/AkseliManninen/lunssi/blob/ac540661151fa00691f97d2a4971d90d51757662/frontend/app/%5Blocale%5D/layout.tsx#L13
And here: https://github.com/AkseliManninen/lunssi/blob/ac540661151fa00691f97d2a4971d90d51757662/frontend/app/%5Blocale%5D/%5Bregion%5D/page.tsx#L32

File: docs/custom-domain.md (hunk 1 above)
Comment (AkseliManninen) [reply docs/custom-domain.md:15]: True! I'll fix.

**Comment by AkseliManninen** [comment]
@yannjor Does the changes in metadata look right to you now?

**Comment by yannjor** [comment]
> @yannjor Does the changes in metadata look right to you now?

Parfait ! 🥖

**Pull request merged**